docker/package/packages.py

```diff
@@ -17,15 +16,15 @@
 # the config
 networks = {
     "mainnet": "mainnet",
-    "jakartanet": "jakartanet",
     "ghostnet": "ghostnet",
+    "limanet": "https://teztnets.xyz/limanet",
     "kathmandunet": "kathmandunet",
 }
 networks_protos = {
-    "mainnet": ["014-PtKathma"],
-    "jakartanet": ["013-PtJakart"],
-    "ghostnet": ["014-PtKathma"],
-    "kathmandunet": ["014-PtKathma"],
+    "mainnet": ["PtKathma"],
+    "ghostnet": ["PtKathma"],
+    "limanet": ["PtLimaPt"],
+    "kathmandunet": ["PtKathma"],
 }
 
```

Comment:
In these file, as part of the node installation definition, `/usr/bin/tezos-node-{network}` are added, which is a script calling `tezos-node` with some adjustments.

I think that this, an in general all of our scripts here, should also switch to use the new names (in this case `tezos-node`) when binaries are called.

The old ones might still work with symlinks just fine, but it's better to deal with this now and cover the name change fully IMO